### PR TITLE
Removed prefect test `test_last_updated()` and added `pytest==7.1.2` to requirements.txt

### LINKED ISSUE #1309: Run unit tests for Prefect
### Overview
We need to be able to run unit tests for Prefect. Currently, `docker-compose run prefect pytest` is failing.

We can follow the unit test infra setup that the `api` is using. Prefect does seem to have a few unit tests, but we cannot use docker to run them right now.

### Action Items
- [x] Add `pytest` to Prefect's requirements.txt file. `pytest` is already in the Pipfile, but we are using requirements.txt in the Dockerfile.
- [x] See if `docker-compose run prefect pytest` works after that

### PR DESCRIPTION
**Pull request by edwinjue**
Fixes #1309

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: server/prefect/requirements.txt [hunk 1]
```diff
@@ -44,3 +44,4 @@ websocket-client==0.58.0
 websockets==8.1
 zict==2.0.0
 zipp==3.4.1
+pytest==7.1.2
```
Comment (nichhk) [inline server/prefect/requirements.txt:47]: Please insert pytest in alphabetical order

**Review by nichhk — CHANGES_REQUESTED** [review]
Thanks Edwin!

I also forgot to note that we should run the prefect tests as part of our continuous integration. Could you add that [here](https://github.com/hackforla/311-data/blob/dev/.github/workflows/build_test_backend.yml#L31)?

File: server/prefect/requirements.txt (hunk 1 above)
Comment (edwinjue) [reply server/prefect/requirements.txt]: ok, done

Comment (edwinjue) [reply server/prefect/requirements.txt]: added a line to run prefect pytest in /.github/workflows/build_test_backend.yml as per your note regarding continuous integration

**Review by nichhk — APPROVED** [review]
Thanks Edwin! Please merge this at your convenience.

**Comment by edwinjue** [comment]
All done

**Pull request merged**